src/harness/testcases/WINNF_FT_S_FAD_testcase.py

```diff
@@ -0,0 +1,376 @@
+            self.assertEqual( grants_of_cbsd[0]['grantExpireTime'], grant_response[index]['grantExpireTime'])
+            self.assertFalse(grants_of_cbsd[0]['terminated'])
+    
+    def generate_FAD_1_default_config(self, filename):
+        """Generates the WinnForum configuration for FAD.1"""
+        # Load device info
+        device_a = json.load(
+            open(os.path.join('testcases', 'testdata', 'device_a.json')))
+        device_b = json.load(
+            open(os.path.join('testcases', 'testdata', 'device_b.json')))
+        device_c = json.load(
+            open(os.path.join('testcases', 'testdata', 'device_c.json')))
+        # Device #2 is Category B
+        # pre-loaded.
+        self.assertEqual(device_b['cbsdCategory'], 'B')
+        conditionals_b = {
+            'cbsdCategory': device_b['cbsdCategory'],
+            'fccId': device_b['fccId'],
+            'cbsdSerialNumber': device_b['cbsdSerialNumber'],
+            'airInterface': device_b['airInterface'],
+            'installationParam': device_b['installationParam']
+        }
+        conditionals = {'registrationData': [conditionals_b]}
+        del device_b['cbsdCategory']
+        del device_b['airInterface']
+        del device_b['installationParam']
+        devices = [device_a, device_b, device_c]
+        # Grants
+        grants = []
+        for index in range(len(devices)):
+            grants.append(json.load(
+             open(os.path.join('testcases', 'testdata', 'grant_0.json'))))
+        # PPAs and PALs
+        ppas = []
+        pals = []
+        pal_low_frequency = 3550000000.0
+        pal_high_frequency = 3560000000.0
+        pal_record = json.load(
+          open(os.path.join('testcases', 'testdata', 'pal_record_0.json')))
+        ppa_record = json.load(
+          open(os.path.join('testcases', 'testdata', 'ppa_record_0.json')))
+        ppa, pal = makePpaAndPalRecordsConsistent(ppa_record,
+                                                                [pal_record],
+                                                                pal_low_frequency,
+                                                                pal_high_frequency,
+                                                                device_a['userId'])
+        ppas.append(ppa)
+        pals.extend(pal)
+        # ESC senosrs
+        esc_sensors = []
+        esc_sensors.append(json.load(
+              open(os.path.join('testcases', 'testdata', 'esc_sensor_record_0.json'))))
+        # SAS Test Harness 
+        sas_harness_config = {
+          'sasTestHarnessName': 'SAS-TH-2',
+          'url': 'https://localhost:9003/v1.2',
+          'serverCert': os.path.join('certs', "sas.cert"),
+          'serverKey': os.path.join('certs', "sas.key")
+        }	
+        config = {
+          'registrationRequests': devices,
+          'conditionalRegistrationData': conditionals,
+          'grantRequests': grants,
+          'ppas': ppas,
```

Comment:
`ppaRecords` -- see the new section in the meeting minutes doc which attempts to collect the names used in other tests so that we can more easily maintain consistency (and refer to FAD.2 for anything that's not fully specified in the doc).

Similar for `escSensorRecords` and `sasTestHarnessConfig` below, plus the SAS TH config object above.

Reply:
I updated those names according to that.

Reply:
Please check the values in `sas_harness_config` -- I think they are slightly off.